process-services/latest/admin/index.md

```diff
@@ -57,7 +57,7 @@ Process Services uses [Logback](https://logback.qos.ch){:target="_blank"} for lo
 
 Process Services installs with the default Logback configuration reading from `<Tomcat install location>/webapps/activiti-app/WEB-INF/classes/logback.xml` and the equivalent location for Process Services Administrator.
 
-The default configuration can be overridden by placing your own `logback.xml` in `<Tomcat install location>/lib`.
+The default configuration can be overridden by placing your own `logback.xml` or  logback-test.xml  in `<Tomcat install location>/lib`.
```

Comment:
```suggestion
The default configuration can be overridden by placing your own `logback.xml` or  `logback-test.xml`  in `<Tomcat install location>/lib`.
```

Reply:
I changed and committed the new files. Please, re-review.